challenge-programs/high-performance-tidb-challenge.md

```diff
@@ -232,32 +232,65 @@ In this period, your team will be working together with the mentor to diagnose p
 
-### Step 7: Present your work
+### Step 6: Present your work
 
-Submit your presentation slides before 10:00, Nov.20 (China Standard Time). The presentation should be no longer than 7 minutes, with the following topics covered:
+Submit your presentation slides before 10:00, Dec.4th (China Standard Time). The presentation should be no longer than 7 minutes, with the following topics covered:
 
 * Introduction to the workload situation before and after your optimization
 * Work allocation among team
 * Performance improvement
 
-The online presentation will be held at 13:00-17:00, Nov.21 (China Standard Time). The challenge results will be announced after the presentation.
+The online presentation will be held at 13:00-17:00, Dec.5th (China Standard Time). The challenge results will be announced after the presentation.
 
 ## Awards
 
-* 1st prize: 7,000 US dollars for the team; 1,350 US dollars for the mentor
-* 2nd prize: 4,250 dollars for the team; 1,350 US dollars for the mentor
-* 3rd prize: 1,350 US dollars for the team
+* 1st prize: 7,000 USD for the team; 1,350 USD for the mentor
+* 2nd prize: 4,250 USD for the team; 1,350 USD for the mentor
+* 3rd prize: 1,350 USD for the team
 
 >**Note**: Teams that finish the final presentation will be evaluated by the judges, among which no more than five winning teams will be generated. The winning teams will be awarded as described above. Each member of the finisher team can get a special swag. If the winning team does not have a mentor, the mentor prize will be automatically added to the team.
 
+### Newly added resources
+
+Considering the difficulty of the tasks and the limited time of the participants, the organizing committee has decided to extend the competition to Nov.29. What's more, the committee has decided to offer more resources for the teams, including benchmark facilities on cloud platforms and TiDB Cloud cash coupons!
+
+**Benchmark facilities on the cloud platforms**
+
+All teams are eligible to apply for the benchmark facilities on the cloud platforms to better experiment and verify ideas. Benchmark facilities for each team include 1 cluster with 6 servers. The specific configuration of a single server is listed as below.
+
+4C8G + data disk 200G + public network bankwidth
+
+***How to apply***
+
+Send an email to zhangyurui@pingcap.com with the subject formatted below.
+Subject: Your team name + application of server resource
+Make sure your Email address is valid as the server information will be sent by email.
+
+**TiDB Cloud cash coupons**
+
+To thank the contribution of participating teams, the organizing committee has prepared TiDB Cloud cash coupons worth 70,000 USD!
```

Comment:
To thank the contribution of participating teams, the organizing committee has prepared TiDB Cloud credit coupons worth 70,000 USD!